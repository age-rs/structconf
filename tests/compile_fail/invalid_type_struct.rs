use structconf::StructConf;

#[derive(StructConf)]
struct Enum(String, i32);

fn main() {}
