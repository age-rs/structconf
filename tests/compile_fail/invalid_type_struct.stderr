error: Cannot #[derive(StructConf)] for the type `unnamed struct`, StructConf is only available for named structs.
 --> $DIR/invalid_type_struct.rs:4:8
  |
4 | struct Enum(String, i32);
  |        ^^^^
